html/global_attributes.json

```diff
@@ -556,7 +556,7 @@
               "version_added": "66"
             },
             "chrome": {
-              "version_added": "66"
+              "version_added": "67"
```

Comment:
review?(@jpmedley)

Reply:
67 for all three flavors of Chrome and 55 for both of Opera.